rtm.go

```diff
@@ -43,16 +43,26 @@ func (api *Client) StartRTMContext(ctx context.Context) (info *Info, websocketUR
 // ConnectRTM calls the "rtm.connect" endpoint and returns the provided URL and the compact Info block.
 //
 // To have a fully managed Websocket connection, use `NewRTM`, and call `ManageConnection()` on it.
-func (api *Client) ConnectRTM() (info *Info, websocketURL string, err error) {
-	return api.ConnectRTMContext(context.Background())
+func (api *Client) ConnectRTM(options ...url.Values) (info *Info, websocketURL string, err error) {
+	return api.ConnectRTMContext(context.Background(), options...)
 }
 
-// ConnectRTM calls the "rtm.connect" endpoint and returns the provided URL and the compact Info block with a custom context.
+// ConnectRTMContext calls the "rtm.connect" endpoint and returns the provided URL and the compact Info block with a custom context.
 //
 // To have a fully managed Websocket connection, use `NewRTM`, and call `ManageConnection()` on it.
-func (api *Client) ConnectRTMContext(ctx context.Context) (info *Info, websocketURL string, err error) {
+func (api *Client) ConnectRTMContext(ctx context.Context, options ...url.Values) (info *Info, websocketURL string, err error) {
 	response := &infoResponseFull{}
-	err = post(ctx, "rtm.connect", url.Values{"token": {api.config.token}}, response, api.debug)
+	var params url.Values
+
+	if len(options) > 0 {
```

Comment:
not a fan of this pattern. functional options are more flexible and tend to be easily to update/deprecate things in the future.
```
type DialConfig struct{}
type DialOption func(*DialConfig)
```